mobile/src/main.jsx

```diff
@@ -23,14 +23,16 @@ import { initClient, initBar, isClientRegistered, resetClient, refreshFolder, on
 import { watchNetworkState, getConnectionType } from './lib/network'
 import { onConnectionChange, setConnectionState } from './actions/network'
 
+import { configureReporter, ANALYTIC_URL, getAnalyticConfiguration } from './lib/crash-reporter'
+
 const loggerMiddleware = createLogger()
 
 const renderAppWithPersistedState = persistedState => {
   const store = createStore(
     filesApp,
     persistedState,
     applyMiddleware(
-      RavenMiddleWare(`https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`),
+      RavenMiddleWare(ANALYTIC_URL, getAnalyticConfiguration()),
```

Comment:
As we use this exact same parameter in `logException` in `crash_reporter.js`, we may refactor this. Check this for inspiration https://github.com/ngokevin/redux-raven-middleware/blob/master/index.es6